.tx/config

```diff
@@ -3,8 +3,8 @@ host     = https://www.transifex.com
 lang_map = ko_KR: ko-KR, pt_PT: pt-PT, pt_BR: pt-BR, ja_JP: ja-JP, zh_CN: zh-CN, zh_HK: zh-HK, zh_TW: zh-TW
 
 [o:ipfs:p:ipld-explorer:r:explore-json]
-file_filter  = public/locales/<lang>/explore.json
-source_file  = public/locales/en/explore.json
+file_filter  = public/locales/<lang>/app.json
+source_file  = public/locales/en/app.json
```

Comment:
use the new namespace files, so we can get translations from ipld-explorer-components (they can still be overridden)